weather-app/app.js

```diff
@@ -0,0 +1,88 @@
+const endpoint = `https://api.openweathermap.org/data/2.5/weather?&units=metric`;
+const myKey = `af06debfaf75258ae5951457ee2bd6f9`;
+const body = document.body;
+const form = document.querySelector("form");
+const input = document.querySelector(".input");
+const app = document.querySelector(".app-card");
+const cardBottom = document.querySelector(".app-card-bottom");
+const temperature = document.querySelector(".temperature-num");
+const weatherIcon = document.querySelector(".weather-icon");
+const weatherStatus = document.querySelector(".weather-status");
+const weahterDescription = document.querySelector(".weather-description");
+const windSpeed = document.querySelector(".wind-speed");
+const windDegrees = document.querySelector(".wind-degrees");
+const clouds = document.querySelector(".clouds");
+const rain = document.querySelector(".rain");
+
+async function appWeather(name = "Hanoi") {
+  const response = await fetch(`${endpoint}&q=${name}&appid=${myKey}`);
+  const data = await response.json();
+  temperature.textContent = data.main.temp;
+  windSpeed.innerHTML = `<div class="weather-info"><i class="fa-solid fa-wind"></i> <span>${data.wind.speed} m/s</span></div>`;
```

Comment:
It's good to have other function build up HTML and pass parameter as response to function

Reply:
handleReponse(wind, rain) {
// do something
}